Jenkinsfile

```diff
@@ -279,16 +302,15 @@ def get_win10_pipeline()
 
                 dir("_build") {
                     try {
+                        bat 'conan remote add conanenter https://center.conan.io'
```

Comment:
It looks like https://center.conan.io has been already added so it causes problems.